discord/voicetwine/egg-pterodactyl-voicetwine.json

```diff
@@ -0,0 +1,152 @@
+{
```

Comment:
what is up with those indenting, it seems wrong / off from the default for both json files